src/components/Collections/Table/Table.vue

```diff
@@ -1,44 +1,44 @@
 <template>
   <div class="flex flex-col mx-2 xsm:mx-4 sm:mx-6 md:mx-8 lg:mx-10 xl:mx-14">
-    <!-- nav bar for table -->
-    <div class="flex flex-col xsm:flex-row justify-between pt-4" v-if="!isTableEmpty">
+    <!-- nav bar for table : table title and search bar -->
+    <div class="flex flex-col xsm:flex-row justify-between pt-4">
       <!-- table title -->
-      <div
-        class="flex flex-row gap-2 text-base sm:text-lg md:text-xl xl:text-2xl font-bold p-2 items-center"
-      >
+      <div :class="tableTitleClass">
         <p>{{ tableTitle }}</p>
-        <p v-if="!pending">({{ totalItemsInTable }})</p>
```

Comment:
we talked about keeping this no?

Reply:
Also if we're going to show the number, it will not be `totalItemsInTable` it will be the total plios that exist overall for the users

Reply:
TODO